.devcontainer/devcontainer.json

```diff
@@ -24,7 +24,16 @@
     "editor.formatOnSave": true,
     "editor.formatOnType": true,
     "[typescript]": {
-      "editor.defaultFormatter": "esbenp.prettier-vscode"
+      "editor.defaultFormatter": "esbenp.prettier-vscode",
+      "editor.codeActionsOnSave": {
+        "source.organizeImports": true
```

Comment:
Personally, I've had very bad experiences with this, super slow and messed up the code more than once when I saved it again while it was still processing.

Reply:
because of `/compatibility` I have to disable it anyway...